Makefile

```diff
@@ -257,10 +260,13 @@ include/sprite/npc/%.h: sprite/npc/%/SpriteSheet.xml tools/gen_sprite_animations
 	@mkdir -p $(shell dirname $@)
 	@echo "building $@"
 	@$(PYTHON) tools/gen_sprite_animations_h.py $@ sprite/npc/$* $(NPC_DIRS)
-include/sprite: $(foreach dir, $(NPC_DIRS), include/$(dir).h)
+
+
+### Linker ###
 
 $(LD_SCRIPT): $(SPLAT_YAML)
-	$(SPLAT) --modes ld
+	$(SPLAT) --modes ld bin Yay0 PaperMarioMapFS PaperMarioMessages img PaperMarioNpcSprites --new
```

Comment:
since this is now splitting out everything required, I think it shouldn't be the LD_SCRIPT target but instead like...split-req or something? open to other names

Reply:
Perhaps. LD_SCRIPT is the primary output - assets only get resplit when needed (with `--new`).

Reply:
Making it a phony target would mean we couldn't depend on it as we do now AFAIK

Reply:
For people who want to manually split you can still do `make split` and it'll route to this rule unconditionally

Reply:
allroyght, we can see how it goes